src/many-cli/src/main.rs

```diff
@@ -45,12 +45,39 @@ enum SubCommand {
 }
 
 #[derive(Parser)]
-struct IdOpt {
-    /// An hexadecimal value to encode, or an identity textual format to decode.
-    arg: String,
+#[clap(override_help(
+    r#"many-id
+
+Transform a textual ID into its hexadecimal value, or the other way around. If the argument is
+neither hexadecimal value or identity, try to see if it's a file, and will parse it as a PEM file
+
+USAGE:
+   many id <ARG> -- [SUBID]
+   many id <MODULE> <SLOT> <KEYID> -- [SUBID]
```

Comment:
I'm not a super-fan of overriding help. Do you think this would be better as a separate command instead (like `hsm-id`)?

Reply:
I'm not a fan either, but I thought it would be unfortunate to create a separate command just for that. 

In the end, you're probably right - it would be simpler and cleaner to have a separate command.